src/System.ComponentModel.TypeConverter/src/System/ComponentModel/LicenseManager.cs

```diff
@@ -303,17 +303,11 @@ private static bool ValidateInternalRecursive(LicenseContext context, Type type,
                 // NOTE : Must look directly at the class, we want no inheritance.
                 //
 
-                //TODO NETSTANDARD2.0
-                // original:                 LicenseProviderAttribute attr = (LicenseProviderAttribute)Attribute.GetCustomAttribute(type, typeof(LicenseProviderAttribute), false);
-
                 IEnumerator attributes = CustomAttributeExtensions.GetCustomAttributes<LicenseProviderAttribute>(type, false).GetEnumerator();
                 LicenseProviderAttribute attr = null;
```

Comment:
Hah -- missed that one :) -- I thought the code was already moved and just the comment remained :).